plugins/base/proto/base.pb.go

```diff
@@ -1,7 +1,7 @@
 // Code generated by protoc-gen-go. DO NOT EDIT.
-// source: github.com/hashicorp/nomad/plugins/base/base.proto
+// source: github.com/hashicorp/nomad/plugins/base/proto/base.proto
```

Comment:
I think we need a way too disable go vet on protobuf generated code, this and the other driver plugin proto generated code always result in go vet warnings because it can't correctly resolve usages.